scripts/nesi_scripts/vm_params_gen.sl

```diff
@@ -54,7 +54,8 @@ end_time=`date +$runtime_fmt`
 echo $end_time
 
 timestamp=`date +%Y%m%d_%H%M%S`
-if [[ -f $OUT_DIR/vm_params.yaml ]]; then
+res=`python $gmsim/qcore/qcore/validate_vm.py params $OUT_DIR/vm_params.yaml`
+if [[ $? == 0 ]]; then
```

Comment:
I personally like saving $? the line below the call, just so someone doesn't put a line before the if statement.

Up to you. Having no space between the if and the `res` line might also be enough.